_config.ts

```diff
@@ -125,28 +143,13 @@ site.use(
   }),
 );
 
-// This is a work-around due to deno-dom's dependency of nwsapi not supporting
-// :has selectors, nor having intention of supporting them, so using `body:not(:has(.ddoc))`
-// is not possible. This works around by adding an `apply-prism` class on pages that
-// don't use a ddoc class.
-site.process([".html"], (pages) => {
-  for (const page of pages) {
-    const document = page.document!;
-    if (!document.querySelector(".ddoc")) {
-      document.body.classList.add("apply-prism");
-      document
-        .querySelectorAll("body.apply-prism pre code")
-        .forEach((element) => Prism.highlightElement(element));
-    }
-  }
-});
-
```

Comment:
Great to see this being removed! Does the highlighting still work on reference pages? I could be wrong but iirc it was used there to add highlighting at some point.

Reply:
As far as I can see, the highlighting is still working in API ref. I do notice the styles are slightly different between local and prod, but I think that might be the same on main... :S